collect-user-data

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
One obvious follow-up question would be "is it worth having bash in the production images?". I'm not entirely convinced the expanded feature set is worth it, and I've kind of used the "I need `bash`isms" as a "hey, think about switching to Javascript" signal in the past, but... all obviously open for discussion.